javaparser-core-generators/src/main/java/com/github/javaparser/generator/NodeGenerator.java

```diff
@@ -53,6 +57,23 @@ public final void generate() throws Exception {
         return new Pair<>(nodeCu, nodeCoid);
     }
 
+    /**
+     * Annotate a method with the {@link Override} annotation, if it overrides other method.
+     *
+     * @param nodeMetaModel     The current meta model.
+     * @param methodDeclaration The method declaration.
+     */
+    protected void annotateWhenOverridden(BaseNodeMetaModel nodeMetaModel, MethodDeclaration methodDeclaration) {
+        Class<? extends Node> type = nodeMetaModel.getType();
+        Class<?> superClass = type.getSuperclass();
+
+        boolean isOverriding = Arrays.stream(superClass.getMethods())
+                .filter(m -> m.getName().equals(methodDeclaration.getNameAsString()))
+                .anyMatch(m -> m.getParameters().length == methodDeclaration.getParameters().size());
+        if (isOverriding)
+            annotateOverridden(methodDeclaration);
+    }
```

Comment:
please do include braces (I mention it here, but it applies to the other PRs too :) )